recursive/recursive_searching.js

```diff
@@ -8,7 +8,24 @@
  * 4. Otherwise, determine which direction the binary search needs to go in
  */
 
-const binary_search = (arr, target, start, end) => {}
+const binary_search = (arr, target, start, end) => {
+    const array = arr.sort((a, b) => a-b)
+    let mid_index = Math.floor((start+end)/2)
+
+    while(start <= end){
```

Comment:
Instead of a while loop here it might be better to put this check within an if statement. Since you're using recursion the while loop wouldn't be needed.